lib/rubocop/cop/layout/empty_line_after_requires.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Layout
+      # Checks for a newline after requires.
+      #
+      # @example
+      #   # bad
+      #   require 'a'
+      #   require 'b'
+      #   foo
+      #
+      #   # good
+      #   require 'a'
+      #   require 'b'
+      #
+      #   foo
+      #
+      # @example EnforcedStyle: beginning_of_file_only (default)
+      #   # good
+      #   foo
+      #
+      #   require 'a'
+      #   bar
+      #
+      # @example EnforcedStyle: all
+      #   # bad
+      #   foo
+      #
+      #   require 'a'
+      #   bar
+      #
+      #   # good
+      #   foo
+      #
+      #   require 'a'
+      #
+      #   bar
+      #
+      # @example RequireMethodNames: ['require', 'require_relative'] (default)
```

Comment:
Will the cop handle properly several `require` groups with blank lines between them? This happens from time to time when the ordering of the requires matters.

Reply:
Yes. Successive `require`s are treated as a single group, and this cop checks for an empty line after each group.

```ruby
require 'a'
require 'b'

foo
```

Another example:

```ruby
require 'a'
require 'b'

require 'c'
require 'd'

require 'e'
require 'f'
```